dist/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard project</title>
+<script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script></head>
+<body>
+    <section class="container">
+        <h1>Leaderboard</h1>
+        <div class="contents">
+
+                <div  class="table-part">
+                    <nav>
+                        <h2>Recent scores</h2>
+                        <button class="refresh">Refresh</button>
+                    </nav>
+                    <table>
+                        <tbody class="tbody">
+
+                        </tbody>
+                    </table>
+                
+                </div>
+                <form action="" class="form">
+                    <h2>Add your score here</h2>
+                    <div class="inputs">
+                        <input type="text" name="" id="name">
+                        <input type="number" id="score">
+                        <input type="submit" value="submit" id="submit">
+                    </div>
```

Comment:
- [ ] [OPTIONAL] Good job so far! Your API is working well but when I click `Submit`, the information I entered in `Inputs` doesn't disappear until I click `Refresh`. It would be great if you could correct this.

#
![image](https://user-images.githubusercontent.com/63915024/181573599-b4de850e-1af6-4832-8fe8-cc0e8cb7d48b.png)
#